tests/test_chromatic.py

```diff
@@ -1313,6 +1313,9 @@ def evaluateAtWavelength(self, wave):
             ret = ret.shear(g1 = this_shear)
             return ret
 
+        def __repr__(self):
+            return 'galsim.ChromaticGaussian(%r)'%self.sigma
+
```

Comment:
I wasn't sure that we needed to test the chromatic caches specifically, but I did add a test for the LRU_Cache itself.  I think this probably sufficient coverage, but let me know if you think I'm omitting a particular failure mode.